selfdrive/car/honda/values.py

```diff
@@ -681,6 +685,13 @@ class CAR:
       b'77959-TXM-A230\x00\x00',
     ],
   },
+  CAR.HRV: {
+    (Ecu.gateway, 0x18daeff1, None): [b'38897-T7A-A010\x00\x00'],
+    (Ecu.eps, 0x18da30f1, None): [b'39990-THX-A020\x00\x00'],
+    (Ecu.fwdRadar, 0x18dab0f1, None): [b'36161-T7A-A240\x00\x00'],
+    (Ecu.srs, 0x18da53f1, None): [b'77959-T7A-A230\x00\x00'],
+    (Ecu.combinationMeter, 0x18da60f1, None): [b'78109-THX-A210\x00\x00'],
```

Comment:
No fwdCamera? Are you sure this is the complete list?

Reply:
these were all the codes under carFw that csouers instructed us.  The car does have a forward camera, but not sure why its not showing up.  When c2 is powered off, the camera works fine with honda lkas.
  
![ecu2](https://user-images.githubusercontent.com/15847404/79802478-0131da00-832e-11ea-8b40-6457950a03e7.png)